ip/event_manager.go

```diff
@@ -102,6 +102,11 @@ func (m *manager) waitTwiceLeaseTime(ctx context.Context) {
 			if !m.isStopping() {
 				return
 			}
+			master, err := m.locker.IsMaster(ctx)
```

Comment:
Don't you want to test if `err != nil `?

Reply:
Nope